fvm/evm/evm.go

```diff
@@ -23,6 +23,10 @@ func StorageAccountAddress(chainID flow.ChainID) (flow.Address, error) {
 	return sc.EVMStorage.Address, nil
 }
 
+func RandomBeaconAddress(chainID flow.ChainID) flow.Address {
+	return systemcontracts.SystemContractsForChain(chainID).RandomBeaconHistory.Address
+}
+
```

Comment:
Does this have to be here, given we pass the chainID to the underlying layer. I know we do this for storage account but I think that's a legacy from the time EVM storage address was not part of system contracts.